Quarterly Update WMS ID: 181

tmcginn opened the pull request:
Fixes required for changes to the base Linux instance.

kaymalcolm reviewed (approved):
Tom, I saw a few files deleted.  Assuming its okay and doesn't impact anything.  data-management-library/database/docker/create-database-in-docker/workshops/livelabs/manifest.json

tmcginn commented:
Thanks @kaymalcolm - yes, I should have noted that I removed a bunch of legacy files. This workshop doesn't run on LiveLabs, so I removed those files.
